src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.cs

```diff
@@ -0,0 +1,257 @@
+                EnumProvider = "SupportedTargetFrameworksEnumProvider",
+                MultipleValuesAllowed = false,
+                DataSource = new()
+                {
+                    Persistence = "ProjectFileWithInterception",
+                    HasConfigurationCondition = false
+                },
+                Metadata =
+                {
+                    new() { Name = PropertyMetadata.VisibilityCondition, Value = "(not (has-evaluated-value \"Application\" \"TargetMultipleFrameworks\" true))" },
+                    new() { Name = PropertyMetadata.SearchTerms, Value = PPR.ApplicationPage_InterceptedTargetFrameworkProperty_SearchTerms },
+                    new() { Name = PropertyMetadata.DependsOn, Value = "Application::TargetMultipleFrameworks" }
+                }
+            };
+
+            StringProperty targetFrameworksProperty = new()
+            {
+                Name = "TargetFrameworks",
+                DisplayName = PPR.ApplicationPage_TargetFrameworksProperty_DisplayName,
+                Description = PPR.ApplicationPage_TargetFrameworksProperty_Description,
+                HelpUrl = "https://go.microsoft.com/fwlink/?linkid=2147236",
+                Category = GeneralCategoryName,
+                DataSource = new()
+                {
+                    Persistence = "ProjectFileWithInterception",
+                    HasConfigurationCondition = false
+                },
+                Metadata =
+                {
+                    new() { Name = PropertyMetadata.VisibilityCondition, Value = "(has-evaluated-value \"Application\" \"TargetMultipleFrameworks\" true)" },
+                    new() { Name = PropertyMetadata.SearchTerms, Value = PPR.ApplicationPage_TargetFrameworksProperty_SearchTerms },
+                    new() { Name = PropertyMetadata.DependsOn, Value = "Application::TargetMultipleFrameworks" }
+                }
+            };
+
+            StringProperty installOtherFrameworksProperty = new()
+            {
+                Name = "InstallOtherFrameworks",
+                DisplayName = PPR.ApplicationPage_InstallOtherFrameworksProperty_DisplayName,
+                Category = GeneralCategoryName,
+                DataSource = new()
+                {
+                    PersistedName = "InstallOtherFrameworks",
+                    Persistence = "ProjectFileWithInterception",
+                    HasConfigurationCondition = false
+                },
+                ValueEditors =
+                {
+                    new()
+                    {
+                        // TODO: EditorType could come from a constant
+                        EditorType = "LinkAction",
+                        Metadata =
+                        {
+                            new() { Name = "Action", Value = "URL" },
+                            new() { Name = "URL", Value = "http://go.microsoft.com/fwlink/?LinkID=287120" }
+                        }
+                    }
```

Comment:
Some helper methods for this kind of thing could be helpful.

This could be written as:

```suggestion
                    HyperlinkActionEditor("http://go.microsoft.com/fwlink/?LinkID=287120")
```